### PR TITLE
Tiny improvements to the help msgs in the search script

### PR DESCRIPTION
**Pull request by gieses**
Dear *,

a tiny PR to enhance the command line arguments and their help msgs in search.py. When executing --help the default values are now also printed.

HTH
Cheers,
Sven

### CODE REVIEW

File: .gitignore
Comment (milot-mirdita) [inline .gitignore:1]: Please remove this file from the commit

File: colabfold/mmseqs/search.py
Comment (milot-mirdita) [inline colabfold/mmseqs/search.py:1]: Please use `formatter_class=argparse.ArgumentDefaultsHelpFormatter`
See: https://stackoverflow.com/a/12151325

Comment (gieses) [reply colabfold/mmseqs/search.py:1]: TIL! Sure, I also added it to the batch.py file, following the import style from the file.

File: .gitignore
Comment (gieses) [reply .gitignore:1]: Sure, I assume you mean the additions to the file.
Note that .idea folders are created by jetbrain's python IDE and are usually good to have in the .gitignore file.

**Comment by gieses** [comment]
Changes result in:
image

**Comment by milot-mirdita** [comment]
Thank you!

**Pull request merged**